toast.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Toast</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header class="nav-header">
+        <div class="nav-header-div">
+            <div class="nav-header-div-lt">
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="index.html">Home</a>
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="docs.html">Docs</a>
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="#">About us</a>
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="#">Blogs</a>
+            </div>
+            <div class="nav-header-div-rt">
+                <a href="https://github.com/eakansh125/Fleet-UI/tree/dev"><img class="margin-lr"
+                        src="https://img.icons8.com/ios-filled/20/000000/github.png" /></a>
+                <a href="#"><img class="margin-lr"
+                        src="https://img.icons8.com/material/20/000000/do-not-disturb-2.png" /></a>
+            </div>
+        </div>
+    </header>
+
+    <div class="view">
+        <aside>
+            <div class="nav-pill">
+                <a href="docs.html">Avatar</a>
+                <a href="badge.html">Badge</a>
+                <a href="alert.html">Alert</a>
+                <a href="button.html">Button</a>
+                <a href="card.html">Card</a>
+                <a href="image.html">Image</a>
+                <a href="input.html">Input</a>
+                <a class="active" href="toast.html">Toast</a>
+            </div>
+        </aside>
+
+        <main class="main-view">
+            <div class="main-view-container">
+                <h1>Toast</h1>
+                <p>Text fields allow users to enter text into a UI. They typically appear in forms and dialogs.</p>
+                <h2>Basic toast</h2>
+                <p>Text fields allow users to enter text into a UI. They typically appear in forms and dialogs.</p>
+                <div class="component-container">
+                    <div class="toast-container red">
+                        <p>This is an alert toast — Click X to dismiss!</p>
+                        <svg class="pointer" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" width="18" height="18" viewBox="0 0 50 50"
+                            style=" fill:#000000;">
```

Comment:
Change- Avoid inline styling.

Reply:
Actually it's a copied svg, but yeah I got your point. Will keep that in check. Thank you!